utils.py

```diff
@@ -1,43 +1,59 @@
 #!/usr/bin/env python
 
 import sys
+import array
 import pygame
 
 import wx
 wx.App()
 
 import numpy as np
 
+from PIL import Image
+
 from skimage.color import rgb2gray
 from skimage.transform import resize
 from skimage.io import imread
+from skimage.util import img_as_float
 
 import matplotlib.pyplot as plt
 import matplotlib.image as mpimg
 
+SRC_W = 615 # 640
+SRC_H = 480
+SRC_D = 3
+OFFSET_X = 0 # 400
+OFFSET_Y = 0 # 240
```

Comment:
Yeah, these are the sizes & offsets for the emulator window on my machine. The emulator runs at 640x480 and in the center of my primary monitor (offset from 0,0 by 400,240). It was easier for me just to not move the window around every time I ran the thing. I'll go ahead and remove them.